### ISSUE TITLE
SNES and Genesis games only display as black screen

### ISSUE THREAD

**Issue by perseusx** [OpenEmu/OpenEmu#1470]
The initial version of OpenEmu ran great for me, but since then I have been unable to play SNES and Genesis roms (others seem to work fine) and I'm hoping someone can help me. 

When I try to run any SNES or Genesis ROM, the game window is black with no image or sound. I've deleted the OpenEmu app and the folder and files from the Application Support folder and re-installed several times, to no avail. The roms run in other emulators. I'm running the latest OSX.9.

Here is the output from my console as this happens:

OpenEmu[7696]: OELibraryDatabase loadFromURL: 'file:///Users/XXXXXXXXX/Library/Application%20Support/OpenEmu/Game%20Library/'
OpenEmu[7696]: Apple Internal Keyboard / Trackpad
OpenEmuHelperApp[7701]: parent application is: OpenEmu
OpenEmuHelperApp[7701]: Some shit about the game.
OpenEmuHelperApp[7701]: loading /Users/XXXXXXXXXX/Desktop/ROM.smc
OpenEmuHelperApp[7701]: No ROM file header found.
OpenEmuHelperApp[7701]: Get filename
OpenEmuHelperApp[7701]: "**>6_**'_**II__3S__**_" [bad checksum] LoROM, 64Mbits, ROM+RAM, NTSC, SRAM:Corrupt, ID:__q_, CRC32:A300E97B
OpenEmuHelperApp[7701]: Setting up emulation
OpenEmu[7696]: SETUP DONE.
OpenEmuHelperApp[7701]: Begin separate thread
OpenEmuHelperApp[7701]: RT policy: 0.007000s (limit 0.030000s) every 0.016639s

**Comment by JamieJ1** [comment]
Hey guys. I seem to be having this same issue. I am trying to run Donkey Kong Country (USA).fsc and seeing only a black screen. Open Emu is also then reluctant to close the game or the app when I try to quit (not sure whether this is related to the issue of the game not running).

I have had to download and install v1.0.4 of open emu as i am not yet running El Capitan. (running 10.10.5). I tried entering the above mentioned command line into Terminal to no avail: **defaults write org.openemu.OpenEmu OEGameCoreManagerModePreference OEDOGameCoreManager**

Does anyone have any other suggestions for making this work?
